### DISCUSSION TITLE
How to install tailwind css ?

### DISCUSSION THREAD

**Discussion by awnaf** [tailwindlabs/tailwindcss#8546] [General]
hello,
how to install tailWind css ?

because this steps dosn't work for me :
https://tailwindcss.com/docs/installation

it's just a css i don't know why they complicate installation

**Comment by Kittonn** [comment]
try this🔥
https://dev.to/slimpython/how-to-install-tailwindcss-via-npm-all-steps-explained-2n5o

**Comment by yeasin2002** [comment]
### You can see this in [Official documentation](https://tailwindcss.com/docs/installation) Here  
### Or If you have any kind of Problem you can Checkout this full [Tailwindcss Installations Notes ](https://github.com/Mdkawsarislam2002/Programming-Resources-And-Notes/tree/TailwindCSS)

🔰 https://github.com/Mdkawsarislam2002/Programming-Resources-And-Notes/tree/TailwindCSS

**Comment by ghost** [comment]
Check the official documentation and if you have a particular problem Google is your friend !
